### ISSUE TITLE
[BUG]ZeroDivisionError: integer division or modulo by zero

### ISSUE THREAD

**Issue by wuast94** [aristocratos/bpytop#204]
**Describe the bug**

At startup i get the error:
ZeroDivisionError: integer division or modulo by zero

**To Reproduce**

Start bpytop

**Expected behavior**

start normaly

**Screenshots**

----

**Info (please complete the following information):**
 - bpytop version: `bpytop -v`
bpytop version: 1.0.50

 - psutil version: `bpytop -v` (version 5.7.0 or above is required)
psutil version: 5.7.3

 - (Linux) Linux distribution and version:
Linux version 4.19.107-Unraid (root@Develop) (gcc version 9.2.0 (GCC)) #1 SMP Thu Mar 5 13:55:57 PST 2020

 - (OSX/FreeBSD) Os release version:
---

 - Terminal used:
terminator

 - Font used:
Ubuntu Regular 11

 - Python version, `python3 --version` (version 3.6 or above is required):
Python 3.9.0

**Additional context**

contents of `~/.config/bpytop/error.log`
```
27/11/20 (01:35:56) | ERROR: Data collection thread failed with exception: integer division or modulo by zero
Traceback (most recent call last):
  File "/usr/lib64/python3.9/site-packages/bpytop.py", line 2600, in _runner
    collector._draw()
  File "/usr/lib64/python3.9/site-packages/bpytop.py", line 2873, in _draw
    CpuBox._draw_fg()
  File "/usr/lib64/python3.9/site-packages/bpytop.py", line 1752, in _draw_fg
    out += (f'{THEME.inactive_fg} ⡀⡀⡀⡀⡀{Mv.l(5)}{THEME.gradient["temp"][min_max(cpu.cpu_temp[0][-1], 0, cpu.cpu_temp_crit) * 100 // cpu.cpu_temp_crit]}{Graphs.temps[0](None if cls.resized else cpu.cpu_temp[0][->
ZeroDivisionError: integer division or modulo by zero
27/11/20 (01:35:57) | WARNING: Exiting with errorcode (1). Runtime 0:00:02
```
(try running bpytop with `--debug` flag if error.log is empty)

hope i get all right :) when you need additional information feel free to ask :)

**Comment by wuast94** [comment]
ok i get it to work, i just had a negativ cpu and mainboard temp. 
i solved the issue by rescan sensors and apply them
now it starts :) 

i let this issue open and let it up to you to close it if its irrelevant for you, when you want to add the possiblility to show negative temps (sure a rare usecase but for some overclockers possible ? dont know) you can use this issue to keep track of it when you handle it this way :) 

so thanks and keep up :)

**Comment by aristocratos** [comment]
@wuast94
Happy to hear you got it working!

I thought I had already fixed the issue with negative temperature values, will take another look at it.

Would you mind posting the output of `python3 -c "import psutil; print(psutil.sensors_temperatures())"` and `sensors`, might help tracking down the exact cause of this still being an issue.

**Comment by wuast94** [comment]
Would like to help on this and will give you the output tomorrow. Do you need the negativ number? Becouse I fixed this and must try to revert this so I have the negative numbers again :)

**Comment by aristocratos** [comment]
@wuast94 
> Do you need the negativ number?

No, just the output as it is. It's the sensor names that I'm interested in.

**Comment by wuast94** [comment]
python3 -c "import psutil; print(psutil.sensors_temperatures())":
```
{'nct6779': [
shwtemp(label='SYSTIN', current=30.0, high=80.0, critical=80.0), 
shwtemp(label='PCH_CPU_TEMP', current=0.0, high=None, critical=None), 
shwtemp(label='CPUTIN', current=32.0, high=80.0, critical=80.0), 
shwtemp(label='AUXTIN0', current=15.0, high=None, critical=None), 
shwtemp(label='AUXTIN1', current=34.0, high=None, critical=None), 
shwtemp(label='AUXTIN2', current=22.0, high=None, critical=None), 
shwtemp(label='AUXTIN3', current=-29.0, high=0.0, critical=None), 
shwtemp(label='SMBUSMASTER 0', current=50.0, high=None, critical=None), 
shwtemp(label='PCH_CHIP_CPU_MAX_TEMP', current=0.0, high=None, critical=None), 
shwtemp(label='PCH_CHIP_TEMP', current=0.0, high=None, critical=None)
]}
```

sensors:
```
nct6779-isa-0290
Adapter: ISA adapter
Vcore:         472.00 mV (min =  +0.00 V, max =  +1.74 V)
in1:             1.28 V  (min =  +0.00 V, max =  +0.00 V)  ALARM
AVCC:            3.31 V  (min =  +0.00 V, max =  +0.00 V)  ALARM
+3.3V:           3.30 V  (min =  +0.00 V, max =  +0.00 V)  ALARM
in4:             1.86 V  (min =  +0.00 V, max =  +0.00 V)  ALARM
in5:             1.03 V  (min =  +0.00 V, max =  +0.00 V)  ALARM
in6:             1.21 V  (min =  +0.00 V, max =  +0.00 V)  ALARM
3VSB:            3.42 V  (min =  +0.00 V, max =  +0.00 V)  ALARM
Vbat:            3.28 V  (min =  +0.00 V, max =  +0.00 V)  ALARM
in9:             0.00 V  (min =  +0.00 V, max =  +0.00 V)
in10:          856.00 mV (min =  +0.00 V, max =  +0.00 V)  ALARM
in11:            1.08 V  (min =  +0.00 V, max =  +0.00 V)  ALARM
in12:            1.70 V  (min =  +0.00 V, max =  +0.00 V)  ALARM
in13:          928.00 mV (min =  +0.00 V, max =  +0.00 V)  ALARM
in14:            1.86 V  (min =  +0.00 V, max =  +0.00 V)  ALARM
Array Fan:        0 RPM  (min =    0 RPM)
fan2:          1156 RPM  (min =    0 RPM)
MB Temp:        +30.0°C  (high = +80.0°C, hyst = +75.0°C)  sensor = thermistor
CPU Temp:       +33.0°C  (high = +80.0°C, hyst = +75.0°C)  sensor = thermistor
AUXTIN0:        +15.0°C    sensor = thermistor
AUXTIN1:        +34.0°C    sensor = thermistor
AUXTIN2:        +22.0°C    sensor = thermistor
AUXTIN3:        -29.0°C  (high =  +0.0°C, hyst =  +0.0°C)  sensor = thermistor
SMBUSMASTER 0:  +54.5°C  
intrusion0:    ALARM
intrusion1:    ALARM
beep_enable:   disabled
```

**Comment by aristocratos** [comment]
@wuast94
Should be fixed in v1.0.51
